### ISSUE TITLE
xpubkey - segwit

### ISSUE THREAD

**Issue by Raito00** [LedgerHQ/ledger-wallet-api#12]
Hi!

I have problem with my ledger nano s ...
I generate xpubkey with api on my segwit accaunt, but on prestashop module it cant get any information out ...
i try also in this site: https://goochain.net/ledger
the same problem :(

So xpubkey not working on segwit accaunt >?

BR
Raitis

**Comment by btchip** [comment]
Yes, it's not working yet. We're monitoring standards to see how to handle that
